src/index.html

```diff
@@ -1,13 +1,322 @@
+          <div class="category__photo-row">
+            <div class="category__image-wrapper category__image-wrapper--size-short">
+              <img
+                class="category__image"
+                src="./images/photos/photo_5.png"
+                alt="Audio sunglasses"
+              >
+            </div>
+            <div class="category__image-wrapper category__image-wrapper--size-long">
+              <img
+                class="category__image"
+                src="./images/photos/photo_6.png"
+                alt="Audio sunglasses"
+                  >
+            </div>
+          </div>
+          <h3 class="category__title">Audio sunglasses</h3>
+        </div>
+        <div class="container container--grid">
+          <a href="#" class="button">View all</a>
+        </div>
+      </section>
+      <section id="why-buy"  class="page__section about">
+        <div class="container">
+          <div class="about__content">
+            <h2 class="about__title">
+              Why buy direct from Bose
+            </h2>
+            <div  class="about__description">
+              <p class="about__subtitle">
+                Free 2-day shipping and returns
+              </p>
+              <p class="about__subtitle">
+              90-day risk-free trial
+              </p>
+              <p class="about__subtitle">
+              World class customer service
+              </p>
+              <p class="about__subtitle">
+              My Bose account management
+              </p>
+              <p class="about__paragraph">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="contacts" class="page__section contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+        </div>
+        <div class="container container--grid">
+          <div class="contact-us__form">
+            <form
+              action="http://localhost:8080/api"
+              method="post"
+              class="form"
+            >
+              <input
+                name="name"
+                type="text"
+                placeholder="Name"
+                class="form-field"
+                required
+                >
+              <input
+                name="email"
+                type="email"
+                placeholder="Email"
+                class="form-field"
+                required
+                >
+              <textarea
+                name="message"
+                placeholder="Message"
+                class="form-field form-field--textarea"
+                required
+              >
+              </textarea>
```

Comment:
```suggestion
              <textarea
                name="message"
                placeholder="Message"
                class="form-field form-field--textarea"
                required
             ></textarea>
```
Remove initial spaces:
![image](https://user-images.githubusercontent.com/86923089/209644372-cf662bb5-24be-4777-84b4-75bff1e2bb08.png)